src/transformStateWithClones.js

```diff
@@ -1,13 +1,41 @@
 'use strict';
 
 /**
- * @param {Object} state
+ * @param {Object} stateCopy
  * @param {Object[]} actions
  *
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const STATE_COPYS = [];
```

Comment:
It's not a constant, it's just a variable as you modify it, so the name should be in camelCase.